modules/ROOT/pages/quickstart.adoc

```diff
@@ -30,14 +30,14 @@ Check xref:overview.adoc#_spark_compatibility[this page] for more information ab
 
 [shell]
 ----
-$SPARK_HOME/bin/spark-shell --jars neo4j-connector-apache-spark_2.12-5.0.1_for_spark_3.jar
+$SPARK_HOME/bin/spark-shell --jars neo4j-connector-apache-spark_2.13-5.2.0_for_spark_3.jar
```

Comment:
I cannot make `{exact-connector-version}` work here and in the subsequent sections... any idea how to rely on the attribute instead of hardcoding the value?

Reply:
In blocks like this there are some values that are substituted by default, but you need to explicitly tell asciidoctor to substitute attributes.

You can do this with `[shell, subs="attributes+"]` - where `attributes` tells asciidoctor to substitute attributes (`{...}`) for their values, and `+` applies the default substitutions (in the case of a code black that includes callouts, for example).

Then you can use `{exact-connector-version}` inside the code block.

Reply:
Thanks a lot! Fixed in 7afc6d9bd97999fa1afd604cdef86cd54501f8dd